src/samples/coap/simple-server.c

```diff
@@ -210,6 +210,7 @@ light_method_get(const struct sol_coap_resource *resource, struct sol_coap_packe
 }
 
 static struct sol_coap_resource light = {
+    .api_version = SOL_COAP_RESOURCE_API_VERSION,
```

Comment:
While you're at it, please change `oicgen.py` so that the API version is set there as well.